releasenotes/notes/fix-checks-from-labels-26b10d9d8952eefb.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Checks configured from docker labels / k8s annotations are no
+    longer overridden in the scheduler.
```

Comment:
These release notes should be understandable by customers so I'd describe the fix in terms of the impact rather than the implementation (i.e. what behavior are we fixing?). For example, customers are unlikely to know what the "scheduler" is or what the effect of "no longer overridden" is.